packages/manager/apps/dedicated/client/app/ip/components/list/list.html

```diff
@@ -295,6 +295,7 @@
                 <!-- + -->
                 <th scope="row">
                     <button
+                        data-ng-if="!(ipBlock.version === IP_TYPE.V6 && ipBlock.type === FAILOVER)"
```

Comment:
Simplify the reading 
```suggestion
                        data-ng-if="ipBlock.version !== IP_TYPE.V6 || ipBlock.type !== FAILOVER"
```